src/plugins/es_ui_shared/__packages_do_not_import__/authorization/components/page_error.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiSpacer, EuiEmptyPrompt, EuiPageContent } from '@elastic/eui';
+import React from 'react';
+import { APP_WRAPPER_CLASS } from '../../../../../../src/core/public';
+import { Error } from '../types';
+
+interface Props {
+  title: React.ReactNode;
+  error: Error;
+  actions?: JSX.Element;
+  centerErrorContent?: boolean;
+}
+
+/*
+ * A reusable component to handle full page errors.
+ * This is based on Kibana design guidelines related
+ * to the new management navigation structure.
+ * In some scenarios, it replaces the usage of <SectionError />.
+ */
+
+export const PageError: React.FunctionComponent<Props> = ({
+  title,
+  error,
+  actions,
+  centerErrorContent,
```

Comment:
WDYT of using a convention of prefixing boolean flags with `is`, `can`, and so on, to make their role clearer? For example, `isCentered` reads more clearly to me as a flag, vs. `centerErrorContent` which sounds more like a function due to the initial verb.

Reply:
👍 agreed - fixed.